packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -24,6 +24,12 @@ const double kMaterialMediumMinMargin = 12;
 /// design 3 spec.
 const double kMaterialExpandedMinMargin = 32;
 
+/// Converts a [NavigationDestination] to a [NavigationRailDestination].
```

Comment:
nit: typedef dartdocs should start with "Signature for", from style guide https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#canonical-terminology.

Reply:
Do you have a suggestion for a sentence? I'm struggling to find something good.

Reply:
```
/// Signature for a builder used by [AdaptiveScaffold.navigationRailDestinationBuilder] that converts a
/// [NavigationDestination] to a [NavigationRailDestination].
```